content/general/concepts/algorithm/terms/euclidean-algorithm/euclidean-algorithm.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: 'Euclidean algorithm'
+Description: 'The Euclidean algorithm is a simple and efficient method for finding the highest common factor (HCF), also known as the greatest common divisor (GCD), of two numbers.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Algorithms'
+  - 'Hcf'
+CatalogContent:
+  - 'paths/computer-science'
+---
+
+The `Euclidean algorithm` is a recursive algorithm that allows us to find the highest common factor (HCF) of two numbers. It is based on the observation that if we have two numbers, say A and B, and if B divides A evenly (leaving no remainder), then B is the HCF of A and B. In other words, the larger number is divisible by the smaller number without any remainder.
```

Comment:
[Nitpick] See my feedback on [this line](https://github.com/Codecademy/docs/pull/2455#discussion_r1226023651). They both share the same problem. 
```suggestion
The `Euclidean algorithm` is a recursive algorithm that allows us to find the highest common factor (HCF) of two numbers. It is based on the observation that if we have two numbers, say A and B, and if B divides A evenly (leaving no remainder), then B is the HCF of A and B. In other words, the larger number is divisible by the smaller number without any remainder.
```